controller/Tools.php

```diff
@@ -45,36 +45,65 @@ class Tools extends SimplePageModule
     public function assessmentActivity()
     {
         $service = $this->getServiceManager()->get(ActivityMonitoringService::SERVICE_ID);
+
+        // Data
         $this->setData('activity_data', $service->getData());
-        $this->setData('reasonCategories', DeliveryHelper::getAllReasonsCategories());
-        $this->setData('completed_assessments_config', [
-            ActivityMonitoringService::OPTION_COMPLETED_ASSESSMENTS_AUTO_REFRESH =>
-                $service->getOption(ActivityMonitoringService::OPTION_COMPLETED_ASSESSMENTS_AUTO_REFRESH),
+        $this->setData('reason_categories', DeliveryHelper::getAllReasonsCategories());
+
+        // Config
+        $this->setData('config', [
+            ActivityMonitoringService::AUTO_REFRESH_INTERVAL =>
+                $service->getOption(ActivityMonitoringService::AUTO_REFRESH_INTERVAL),
         ]);
+
         $this->setView('Tools/assessment_activity.tpl');
     }
 
     /**
-     * Get assessment activity data
+     * Show assessment activity data as json
      */
-    public function deliveriesActivityData()
+    public function assessmentActivityData()
     {
-        $this->returnJson(new DeliveriesActivityDatatable());
+        $service = $this->getServiceManager()->get(ActivityMonitoringService::SERVICE_ID);
+        $data = $service->getData();
+
+        // todo - optimize these calls (only need one call to getData())
+
+        // Assessment Activity Data
+        $assessmentActivityData = $data;
+
+        // deliveries activity data
+        $deliveriesActivity = new DeliveriesActivityDatatable($data);
+
+        // completed assessments data
+        $interval = $this->getRequestParameter('interval');
+        $completedAssessments = $this->completedAssessmentsData($interval);
+
+        $this->returnJson([
+            'success' => true,
+            'data' => [
+                'assessment_activity' => $assessmentActivityData,
+                'deliveries_activity' => $deliveriesActivity,
+                'completed_assessments' => $completedAssessments
+            ]
+        ]);
     }
 
     /**
      * Get completed assessments data
      */
-    public function completedAssessmentsData()
+    private function completedAssessmentsData($param)
```

Comment:
`$param` is not good name for parameter. 
Also I don't see the reason why you need to pass it. Controller has access to all the request parameters.